### DISCUSSION TITLE
provide a folder for assets referenced in package's README.md

### DISCUSSION THREAD

**Discussion by microrache** [verdaccio/verdaccio#2853] [Ideas]
It would be cool to have something like a "public"- or "assets"-folder which is exposed by Verdaccio's web server. This folder then could be mounted as a Docker Volume, where e.g. screenshots and other assets are stored. These assets then could be referenced in the package's README.md like 
`![See in action](/-/static/screenshots/my-screenshot.png)`

It already worked in the described way in Verdaccio 4.x by mounting a Docker Volume to:
`/opt/verdaccio/node_modules/@verdaccio/ui-theme/static/screenshots`

But since v5 there seems to be no static-folder or anything comprehensible, which can be used for the desired purpose. Is it possible to re-implement such a directory?

**Comment by juanpicado** [comment]
I like the idea 💡, not sure how could be used now in Verdaccio 5 but worth to keep it on mind for next major.
